### PR TITLE
 Remove jstatus class var

### LINKED ISSUE #67: Find better name for j_status var
rename the var `j_status` to better name, like `test_status` or something better :rocket:

### PR DESCRIPTION
**Pull request by MalloZup**
fix #67 , and adress partially #66

### CODE REVIEW

File: gitarro.rb [hunk 1]
```diff
@@ -30,6 +30,3 @@
   break if b.reviewed_pr_test(comm_st, pr)
 end
 STDOUT.flush
-
-# red balls for jenkins
```
Comment (MalloZup) [inline gitarro.rb]: this is not needed since we exit directly in backend lib

Comment (MalloZup) [reply gitarro.rb]: https://github.com/openSUSE/gitarro/pull/70/files#diff-8b4041b10595d6631a5e220c1cbeb548R122

**Comment by MalloZup** [comment]
```Finished in 1 minute 40.78 seconds (files took 0.20969 seconds to load)
7 examples, 0 failures
``` :sunglasses:

**Comment by MalloZup** [comment]
Imho, we  have to assume that everybody run ``rake ``` locally, and this is fine.  (if we need credentials for unit-tests, then this is a rspec tests)

File: lib/gitarro/backend.rb [hunk 2]
```diff
@@ -7,6 +7,35 @@
 require_relative 'opt_parser'
 require_relative 'git_op'
 
+# this module have helper methods for changelog tests
+# it will be removed soon, but it helps to extract all
+# changelog code from backend class
+module ChangelogTests
```
Comment (juliogonzalez) [inline lib/gitarro/backend.rb]: Nicely done. However this is not only fixing #67 but it is also work for #66 :-)

File: lib/gitarro/backend.rb [hunk 3]
```diff
@@ -83,13 +114,12 @@ def open_newer_prs
...
 
-  # public for etrigger the test
+  # public for retrigger the test
   def retrigger_check(pr)
     return unless retrigger_needed?(pr)
     create_status(pr, 'pending')
     exit 1 if @check
-    launch_test_and_setup_status(pr)
-    j_status == 'success' ? exit(0) : exit(1)
+    launch_test_and_setup_status(pr) == 'success' ? exit(0) : exit(1)
```
Comment (juliogonzalez) [inline lib/gitarro/backend.rb]: For the future I really think we should move all exits to gitarro.rb

**Comment by juliogonzalez** [comment]
I really like optimistic people :-)

But my experience tells me that we should run tests in all cases. People tends to forget about testing, specially after they make "minor" changes to a PR (and that's valid for me as well!)

That said, I agree with your reasoning that if we need credentials then it's not unit testing, so happy to approve & merge as long as we have an issue for this (doesn't need to be critical priority).

File: lib/gitarro/backend.rb (hunk 2 above)
Comment (MalloZup) [reply lib/gitarro/backend.rb]: yes, it is a small change. :)

File: lib/gitarro/backend.rb (hunk 3 above)
Comment (MalloZup) [reply lib/gitarro/backend.rb]: moving the exit in the `bin` cmdline application, can cause that we need to expose maybe other functions to public for exiting, and make some other checks.

I can agree that having all exit in one place ( the commandline.rb) it is better.

**Comment by juliogonzalez** [comment]
LGTM

**Comment by MalloZup** [comment]
issue for moving the tests here:
https://github.com/openSUSE/gitarro/issues/74

**Pull request merged**